src/runtime_src/hip/xrt_hip.h

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+#ifndef xrthip_h
+#define xrthip_h
+
+#include "hip/hip_runtime_api.h"
+
+enum hipModuleDataType {
+  hipModuleDataFilePath = 0,
+  hipModuleDataBuffer
+};
```

Comment:
class enum?

Reply:
this header file is shipped to compile with HIP applications and all the other HIP headers are C based, so using C style for the headers that we ship. Also IREE wants to compile their applications using C compiler